### ISSUE TITLE
Correct way to map escape to left control

### ISSUE THREAD

**Issue by mrlasers** [rvaiya/keyd#224]
I can't for the life of me figure out the proper way to map the left control key as escape. I'm trying to have the escape key perform capslock, capslock key perform control, and left control key perform escape (as God indented).

With a previous version, the following config worked great:

```
[ids]

*

[main]

esc = capslock
capslock = overload(C, leftcontrol)
leftcontrol = esc
```

However, every attempt I've made to get this same behavior in the current version has resulted in "escape down" and "escape up" firing constantly until I release the capslock (leftcontrol) key.

My current config is as follows, although I've also tried various forms of assigning control directly to capslock with the same results:

```
[ids]

*

[main]

capslock = layer(capslock)
esc = capslock
leftcontrol = esc

[capslock:C]

```

**Comment by rvaiya** [comment]
The second config you've posted should do what you describe (it does on my machine). Are you saying `capslock` produces `escape` using your second config? What is the output of `keyd -m`? Are you running any other remapping software (e.g xcape)?

**Comment by mrlasers** [comment]
Other remapping software was totally it! Just not Linux remapping software.

This is happening on Linux Mint hosted in a VM on windows... where I've got the capslock key remapped to control. For some reason, that's causing it to fire the `leftcontrol` down/up nonstop when I hold down capslock in the VM, which turned into `esc` when I remapped it with keyd. Turning off the key manager in power toys completely fixed it.

Thanks for your help.
